test/integration.test.js

```diff
@@ -72,7 +72,7 @@ test('integration v1', async (t) => {
 
   await fastify.ready()
 
-  t.same(
+  t.has(
```

Comment:
why this change?

Reply:
When you retrieve the stored secrets, they will come with an extra field named refresh, which is a Symbol reference. Using same is not working anymore because it check a one to one equality, with has the extra field will not be a problem, but your requested secret values still has to be present, so the test still will be valid.

Reply:
ok makes sense, but do you think we need to return also that symbol then? or maybe filter it out